src/box/replication.cc

```diff
@@ -376,7 +376,6 @@ replica_set_id(struct replica *replica, uint32_t replica_id)
 	say_info("assigned id %d to replica %s",
 		 replica->id, tt_uuid_str(&replica->uuid));
 	replica->anon = false;
-	box_update_replication_synchro_quorum();
```

Comment:
Please, do the same for `replica_clear_id()`. Otherwise it is confusing, why lowering the quorum via clearing the ID is allowed outside of DDL, but increasing the quorum in `set_id()` is not.